File: LAPORAN PRATIKUM/MODUL6/CODEBLOKS DAN PYTHON/SOAL 1/PRAK501-2210817210014-MuhammadQoharyRawidafany.py
def MaxBilangan(q,w,e,r):
    q = int(q); w = int(w); e = int(e); r = int(r)
    if (q>w and q>e and q>r):
        return q
    elif (w>q and w>e and w>r):
        return w
    elif (e>q and e>w and e>r):
        return e
    elif (r>q and r>w and r>e):
        return r
a,b,c,d= map(int,input().split())
hasil = MaxBilangan(a,b,c,d)
print(hasil)
    

File: LAPORAN PRATIKUM/MODUL6/CODEBLOKS DAN PYTHON/SOAL 1/PRAK501-2210817210014-MuhammadQoharyRawidafany.c
#include <stdio.h>
int MaxBilangan (int q, int w, int e, int r){
    if (q>w && q>e && q>r){
        return q;
    }
    else if (w>q && w>e && w>r){
        return w;
    }
    else if (e>q && e>w && e>r){
        return e;
    }
    else if (r>q && r>w && r>e){
        return r;
    }
}
int main()
{
    int a,b,c,d;
    for (int i=1;i<=3;i++){
    scanf("%d %d %d %d", &a, &b, &c, &d);
    int hasil = MaxBilangan(a, b, c, d);
    printf("%d", hasil);
    }
    return 0;
}
